github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-postsubmits.yaml

```diff
@@ -57,3 +57,34 @@ postsubmits:
           resources:
             requests:
               memory: "8Gi"
+
+    - name: publish-fedora-realtime-container-image
+      annotations:
+        testgrid-create-test-group: "false"
+      always_run: false
+      run_if_changed: "cluster-provisioning/images/vm-image-builder/fedora-realtime/.*"
+      decorate: true
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-kubevirtci-quay-credential: "true"        
+      cluster: prow-workloads
+      spec:
+        containers:
+          - image: quay.io/kubevirtci/golang:v20210316-d295087
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/bash"
+              - "-ce"
+              - |
+                cat $QUAY_PASSWORD | docker login --username $(<$QUAY_USER) --password-stdin quay.io
+                ./cluster-provisioning/images/vm-image-builder/create-containerdisk.sh fedora-realtime
+                ./cluster-provisioning/images/vm-image-builder/publish-containerdisk.sh fedora-realtime quay.io/kubevirt/fedora-realtime-container-disk:latest
```

Comment:
```suggestion
                ./cluster-provision/images/vm-image-builder/create-containerdisk.sh fedora-realtime
                ./cluster-provision/images/vm-image-builder/publish-containerdisk.sh fedora-realtime quay.io/kubevirt/fedora-realtime-container-disk:latest
```

Reply:
Running the command locally to confirm that it does not have any more typo(s)... :smile:

Reply:
Command successful. I have pushed the correct changes now.